packages/docusaurus-theme-classic/src/theme/MDXComponents/Img/index.tsx

```diff
@@ -11,8 +11,8 @@ import type {Props} from '@theme/MDXComponents/Img';
 
 import styles from './styles.module.css';
 
-function transformImgClassName(className?: string): string {
-  return clsx(className, styles.img);
+function transformImgClassName(className?: string): string | undefined {
+  return !className ? undefined : clsx(className, styles.img);
```

Comment:
this is not the same as before: we want the `styles.img` to **always** be applied, even if the className prop is undefined

Img is not the same case as Ul which is kind of an exception